projects/desktop/src/index.tsx

```diff
@@ -37,19 +38,44 @@ i18n
     },
   })
 
+const Loader = <Fragment />
+
 // Initiating redux store
 const store = configureStore()
 
+// This component will read the hydrated theme from redux
+interface IOwnProps {
+  children: object,
+  mode?: ITheme,
+}
+type IProps = IOwnProps
+const mapStateToProps = ({themeReducer: {mode}}: IReduxState): ITheme => ({mode})
```

Comment:
And if there are more states from reducers to return? The ITheme won't be enough so it's maybe better IStoreProps but maybe we already talked about it

Reply:
I would do that:
`const mapStateToProps = ({themeReducer: {mode}}: IReduxState): IStoreProps => ({mode})`

There is `IReduxState` because we get the themeReducer from it and
there is `IStoreProps` because we are getting this state from the reducer via the props

Reply:
And with `IStoreProps` instead of `ITheme`, we can get multiple stored props and not only the theme